ukbb_qc/release/vcf_repackage.py

```diff
@@ -39,69 +40,102 @@ def main(args):
     Required for return of results to UKBB.
 
     .. note::
-        Run this script in the same directory as `ukbb_header_reformat.sh`.
+        - Run this script in the same directory as `ukbb_header_reformat.sh`.
+        - This script needs a more recent version of the gnomad methods repo than the VCF export code as it relies on `parallel_file_exists`.
     """
     data_source = "broad"
     freeze = args.freeze
     tranche_data = (data_source, freeze)
     output_dir = args.output_dir
-    coordinates_output_dir = "gs://broad-ukbb/broad.freeze_7/temp"
+    coordinates_output_dir = "gs://broad-ukbb/broad.freeze_7/temp/coordinates"
 
     logger.info("Getting VCF shards...")
-    rg = get_reference_genome("GRCh38")
-    contigs = rg.contigs[:24]  # autosomes + X/Y
+    contigs = []
```

Comment:
Why did you have to change the code to get the contigs this way?

Reply:
I forgot that `get_reference_genome` operates on loci/intervals: https://github.com/broadinstitute/gnomad_methods/blob/master/gnomad/utils/reference_genome.py#L123